en/components/navbar.md

```diff
@@ -188,6 +188,43 @@ If the sample is configured properly, you should see the following in your brows
 </div>
 <div class="divider--half"></div>
 
+#### Add Custom Title
+
+If we want to provide a custom content for a navbar's title, we can achieve this by using `igx-navbar-title` or `igxNavbarTitle` directives. They will replace the default navbar's title provided by `title` input property. The sample below has a custom title with div and img nodes inside:
```

Comment:
Technically these are two selectors of the same directive.

```suggestion
If we want to provide a custom content for a navbar's title, we can achieve this by using `igx-navbar-title` or `igxNavbarTitle` directive. They will replace the default navbar's title provided by `title` input property. The sample below has a custom title with div and img nodes inside:
```

Reply:
Fixed